testing/tests/DevExpress.ui.widgets.editors/validationGroup.tests.js

```diff
@@ -57,6 +57,22 @@ QUnit.module('General', {
         assert.ok(validator.validate.calledOnce, 'Validator should be validated as part of group');
     });
 
+    QUnit.test('should find group if validator is child of validationGroup ', function(assert) {
```

Comment:
Actual test name is too common to me. Who should find? Why we check it with this test? Besides it, this test checks the scenario when validator and validationGroup are created on the same element, not when validator is a child of a ValidationGroup.

Also it's a good idea to add a ticked ID as a suffix of a test name. It can make future debug easier.

```suggestion
    QUnit.test('validator should work a part of a validation group if they are created on the same element (T1102012)', function(assert) {
```